v4/export/sql.go

```diff
@@ -433,8 +440,12 @@ func ShowMasterStatus(db *sql.Conn) ([]string, error) {
 	return oneRow, nil
 }
 
-// GetSpecifiedColumnValue get columns' values whose name is equal to columnName
-func GetSpecifiedColumnValue(rows *sql.Rows, columnName string) ([]string, error) {
+// GetSpecifiedColumnValueAndClose get columns' values whose name is equal to columnName and close the given rows
+func GetSpecifiedColumnValueAndClose(rows *sql.Rows, columnName string) ([]string, error) {
```

Comment:
we might also ToUpper(columnName) when comparing

Reply:
addressed in https://github.com/pingcap/dumpling/pull/273/commits/69c1e7c6012590500e0deceed11c876b49c90f19